djreact/settings.py

```diff
@@ -14,10 +14,11 @@
     "rack-city-dev.herokuapp.com",
     "rack-city-staging.herokuapp.com",
     "rack-city-prod.herokuapp.com",
+    "rack-city-mobile.herokuapp.com",
     "127.0.0.1",
     "localhost",
-    "192.168.1.17",
-    "192.168.0.18",
+    # "192.168.1.17",
```

Comment:
commented these out since there seemed to be some weird routing thing going on with heroku if I kept them in